packages/gatsby/src/utils/source-nodes.ts

```diff
@@ -50,20 +50,21 @@ function warnForPluginsWithoutNodes(state: IGatsbyState, nodes: Node[]): void {
 function getStaleNodes(state: IGatsbyState, nodes: Node[]): Node[] {
   return nodes.filter(node => {
     let rootNode = node
+    let next: Node | undefined = undefined
```

Comment:
nit: undefined is the default, so you shouldn't need the assignment here

Reply:
You're right, I prefer to be explicit :)